src/utils/eventTable.tsx

```diff
@@ -74,10 +74,16 @@ export const getCourts = (family: TFamilyPublic): string | null =>
     .join(" / ") || null;
 
 const getFamilyEvents = (family: TFamilyPublic): TEventWithDocument[] =>
-  [
-    ...family.events.map((event) => ({ event })),
-    ...family.documents.map((document) => document.events.map((event) => ({ event, document }))).flat(1),
-  ].flat(1);
+  Array.from(
```

Comment:
This is a pretty gnarly bit of code, personally I'd prefer to break down the solution to make it easier to read and understand what is going on and why we are doing it (captured in code).